umm-spec-lib/src/cmr/umm_spec/test/umm_record_sanitizer.clj

```diff
@@ -35,13 +35,101 @@
+                                     "OPENDAP DATA (DODS)"
+                                     "OPENDAP DIRECTORY (DODS)"
+                                     "OpenSearch"
+                                     "SERF"
+                                     "SOFTWARE PACKAGE"
+                                     "SSW"
+                                     "SUBSETTER"
+                                     "THREDDS CATALOG"
+                                     "THREDDS DATA"
+                                     "THREDDS DIRECTORY"
+                                     "WEB COVERAGE SERVICE (WCS)"
+                                     "WEB FEATURE SERVICE (WFS)"
+                                     "WEB MAP FOR TIME SERIES"
+                                     "WEB MAP SERVICE (WMS)"
+                                     "WORKFLOW (SERVICE CHAIN)"]}
+   "VisualizationURL" {"GET RELATED VISUALIZATION" ["GIBS" "GIOVANNI" "MAP"]}
+   "CollectionURL" {"DATA SET LANDING PAGE" []
+                    "DOI" []
+                    "EXTENDED METADATA" []
+                    "PROFESSIONAL HOME PAGE" []
+                    "PROJECT HOME PAGE" []}
+   "PublicationURL" {"VIEW RELATED INFORMATION" ["ALGORITHM THEORETICAL BASIS DOCUMENT"
+                                                 "CALIBRATION DATA DOCUMENTATION"
+                                                 "CASE STUDY"
+                                                 "DATA QUALITY"
+                                                 "DATA USAGE"
+                                                 "DELIVERABLES CHECKLIST"
+                                                 "GENERAL DOCUMENTATION"
+                                                 "HOW-TO"
+                                                 "PI DOCUMENTATION"
+                                                 "PROCESSING HISTORY"
+                                                 "PRODUCTION VERSION HISTORY"
+                                                 "PRODUCT QUALITY ASSESSMENT"
+                                                 "PRODUCT USAGE"
+                                                 "PRODUCT HISTORY"
+                                                 "PUBLICATIONS"
+                                                 "RADIOMETRIC AND GEOMETRIC CALIBRATION METHODS"
+                                                 "READ-ME"
+                                                 "RECIPE"
+                                                 "REQUIREMENTS AND DESIGN"
+                                                 "SCIENCE DATA PRODUCT SOFTWARE DOCUMENTATION"
+                                                 "SCIENCE DATA PRODUCT VALIDATION"
+                                                 "USER FEEDBACK"
+                                                 "USER'S GUIDE"]}
+   "DataCenterURL" {"HOME PAGE" []}
+   "DataContactURL" {"HOME PAGE" []}})
+
+(defn valid-types-for-url-content-type
```

Comment:
Use `defn-` to define private functions that are only called within the same namespace. This applies to other functions in this PR as well.

Reply:
done